feature/posts/src/commonMain/kotlin/com/tunjid/heron/posts/PostsViewModel.kt

```diff
@@ -83,22 +100,48 @@ class ActualPostsViewModel(
         },
     )
 
-private fun Flow<Action.SnackbarDismissed>.snackbarDismissalMutations(): Flow<Mutation<State>> =
-    mapToMutation { action ->
-        copy(messages = messages - action.message)
-    }
-
 private fun postsMutations(
     request: PostsRequest,
-    scope: CoroutineScope,
     postsRepository: PostRepository,
-): Flow<Mutation<State>> = flow {
-    when (request) {
-        is PostsRequest.Saved -> {
-            // TODO: Fetch Saved posts
-        }
-        is PostsRequest.Quotes -> {
-            // TODO: Fetch quotes
+): Flow<Mutation<State>> = when (request) {
+    is PostsRequest.Saved -> {
+        // TODO: Fetch Saved posts - return empty for now
+        flowOf(mutationOf { copy(posts = emptyTiledList()) })
+    }
+    is PostsRequest.Quotes -> {
```

Comment:
This wont quite work. These should be paginated, and there's no pagination going on at the moment.  Also `PostRepository.quotes` should be updated to return `Flow<CursorList<TimelineItem>>` and not `Flow<CursorList<Post>>`

Reply:
For pagination, you will need to use `tilingMutations`. Examples include:

* [Timeline](https://github.com/tunjid/heron/blob/main/ui/timeline/src/commonMain/kotlin/com/tunjid/heron/timeline/state/Timeline.kt#L114)
* [Search](https://github.com/tunjid/heron/blob/main/feature/search/src/commonMain/kotlin/com/tunjid/heron/search/SearchViewModel.kt#L475)